client/app/common/translations/Messages_fr_FR.xml

```diff
@@ -578,6 +578,12 @@
    <translation id="common_field_error_maxlength" qtlid="472372">Veuillez saisir un maximum de {{maxlength}} caractères.</translation>
    <translation id="common_field_error_pattern" qtlid="472385">Veuillez saisir un format valide.</translation>
 
+   <translation id="common_stepper_optional_label">(facultatif)</translation>
+   <translation id="common_stepper_modify_this_step">Modifier cette étape</translation>
+   <translation id="common_stepper_skip_this_step">Ignorer cette étape</translation>
+   <translation id="common_stepper_next_button_label">Suivant</translation>
+   <translation id="common_stepper_submit_button_label">Envoyer</translation>
```

Comment:
Are you sure it's the most common label "Envoyer"? Not "Modifier" or something like that?

Reply:
I think "Envoyer" is the most generic term in French. I agree, there is not perfect solution here.